test/lib/jsonStathamSpec.js

```diff
@@ -8,10 +8,11 @@ class TestAdaptor extends AjaxAdaptorBase {
   serverBase() { return 'http://test.com' }
 }
 
+let server
```

Comment:
I added a few tests for `PATCH` and grouped some tests of specific methods together under new `describe` calls